README.md

```diff
@@ -2,24 +2,214 @@
+  <td>IBM (Josh Hursey)**</td>
+  <td>Mellanox (Artem Polyakov)</td>
+ </tr>
+ <tr>
+  <td>UnivTenn-Knoxville (Aurelien Bouteiller)</td>
+  <td>LANL (Howard Pritchard)</td>
+  <td>ORNL (Thomas Naughton)</td>
+ </tr>
+ <tr>
+    <td>Stony Brook Univ (Anthony Curtis)</td>
+    <td>Fujitsu Limited (Takahiro Kawashima)</td>
+    <td>ARM (Dirk Schubert)</td>
+</tr>
+<tr>
+    <td>Altair (Mike Karo)</td>
+    <td>RogueWave (John DelSignore)</td>
+    <td>Sylabs (Geoffroy Vallee)</td>
+</tr>
+</table>
+</div>
+\*  Co-Chair (even year) &nbsp;&nbsp;&nbsp;&nbsp; \** Co-Chair (odd year)
+
+### General voting rules
+
+The philosophy and goal of the ASC is to reach consensus on all, if not most, issues. In the event that full consensus cannot be reached, the ASC will use the following rules:
+
+* Each active ASC Member organization will receive one vote. All votes are required to pass with 2/3 majority. A quorum (>50%) of Members must be present (either physically or virtually). Abstentions will not be counted in the total votes when calculating whether there is a 2/3 majority.
+
+* A Member may submit a vote _in absentia_ to the ASC Co-Chairs in advance of the meeting if attendance by a representative is not possible (e.g., a quarterly meeting that falls during a national holiday period), but each ASC Member organization can only cast one vote. This prevents a Member from voting on behalf of its own entity and acting as a proxy for another member. _In absentia_ votes are counted by the Co-Chairs as if the member were present (thus counting towards the meeting's quorum requirement), but are recorded as having been voted _in absentia_ in the official tally.
+
+Formal votes may, if requested by at least one Member, be conducted as "secret ballots" to abate concerns over customer and/or competitor perceptions. All votes are tallied by the Co-Chairs and recorded in the minutes of the meeting.
+
+### Release Managers
+
+One or more Release Managers are assigned per release series of the PMIx Standard. Coordination and scheduling of releases in the series is delegated to the Release Manager(s) by the ASC, subject to guidance regarding needs on the part of Members. Release Managers have the following duties:
+
+* For each release, lead the establishment of release criteria by the ASC. This can include a list of proposed modifications to be decided upon prior to release and target release dates
+* Shepherd progress towards completing the schedule through interactions with proposal authors
+* Maintain the Revisions section of the document to ensure accurate communication of what has changed in each release
+
+Note that Release Managers are not precluded from proposing modifications to the Standard (including to the series under their management) during the course of their assignment. In addition, Release Managers are not responsible for promoting acceptance of any proposal nor contributing to the completion of a proposal in order to meet schedule.
+
```

Comment:
Should we grant some liberty to the ASC Co-Chairs and Release Managers to work through any merge conflicts that might arise after a quarterly meeting?

As long as there are no significant issues in the merge they should be able to resolve this without more process - using their discretion. If there is a significant issue then they can bring it to the community with a suggested resolution.

Reply:
😆 I guess I continue to mistakenly assume at least a minimal level of common sense is being applied. Sure, we can spell it out. Just remember: the more you try to over-think the process and make everything "rigid", the more corner-cases you create that challenge anyone's ability to apply common sense.

Reply:
Here is what I changed it to:

> Note that Release Managers are not precluded from proposing modifications to the Standard (including to the series under their management) during the course of their assignment. In addition, Release Managers are not responsible for promoting acceptance of any proposal nor contributing to the completion of a proposal in order to meet schedule, and may resolve any merge conflicts resulting from commit of approved changes without further process.